src/StarsRatingBar/test/StarsRatingBar.spec.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { createRendererWithUniDriver, cleanup } from '../../../test/utils/unit';
+
+import StarsRatingBar from '../StarsRatingBar';
+import { starsRatingBarPrivateDriverFactory } from './StarsRatingBar.private.uni.driver';
+
+describe(StarsRatingBar.displayName, () => {
+  const render = createRendererWithUniDriver(
+    starsRatingBarPrivateDriverFactory,
+  );
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('should render stars rating bar', async () => {
+    const { driver } = render(<StarsRatingBar value={1} />);
+
+    expect(await driver.exists()).toBe(true);
+  });
+
+  describe('rate caption', () => {
+    it('should display the correct rate caption', async () => {
+      const rateCaptions = ['bad', 'not good', 'ok', 'good', 'excellent'];
+      const { driver } = render(
+        <StarsRatingBar value={3} rateCaptions={rateCaptions} />,
+      );
+
+      expect(await driver.getDisplayedRateCaptionLabel()).toEqual('ok');
+    });
+
+    it('should not display a rate caption', async () => {
+      const { driver } = render(<StarsRatingBar value={2} />);
+
+      expect(await driver.isRatingCaptionExists()).toBeFalsy();
```

Comment:
Never use toBeFalsy unless you intentionally expect `null` or `undefined`
```suggestion
      expect(await driver.isRatingCaptionExists()).toBe(false);
```